src/lib.rs

```diff
@@ -280,6 +278,7 @@ macro_rules! wrap_pymodule {
 /// If you need to handle failures, please use [Python::run] directly.
 ///
 #[macro_export]
+#[cfg(feature = "macros")]
```

Comment:
I'm sorry for my *lazy* review but could you please add this definition?
```rust
#[macro_export]
#[cfg(not(feature = "macros"))]
macro_rules! py_run {
    ($py:expr, $($val:ident)+, $code:expr) => {{
        pyo3::py_run_impl!($py, $($val)+, &pyo3::unindent::unindent($code))
    }};
}
```

`unindent` is a really small dependency (see [its Cargo.toml](https://github.com/dtolnay/indoc/blob/master/unindent/Cargo.toml)) and I think it's OK.

Reply:
then `py_run` would behave differently depending on the feature? that seems confusing...

Reply:
That would mean that code like

```rust
let code = "asdf";
py_run!(py, a b, code);
```

Would compile fine with `feature = "macros"` disabled, but will no longer compile when it gets enabled.

Reply:
> then py_run would behave differently depending on the feature?

No, it behaves the same.
But
1. with `feature="macros"` and
2. string literal is passed like `py_run!(py, a, "print(a)")`
it works faster thanks to [indoc](https://github.com/dtolnay/indoc).